Documentation/policy/language.rst

```diff
@@ -1228,3 +1228,25 @@ policies to a particular cluster.
 .. only:: epub or latex
 
         .. literalinclude:: ../../examples/policies/kubernetes/clustermesh/cross-cluster-policy.yaml
+
+Clusterwide Policies
+--------------------
+
+`CiliumNetworkPolicy` only allows to bind a policy restricted to a particular namespace. There can be situations
+where one wants to have a cluster scoped effect of the policy, this can be done using cilium's
+`CiliumClusterwideNetworkPolicy` kubernetes custom resource. The specification of the policy is same as that
+of `CiliumNetworkPolicy` with the only difference that it is not namespaced.
```

Comment:
with the only difference that it is not namespace --> except that it is not namespaced.